bundles/org.openhab.core/src/test/java/org/openhab/core/library/types/HSBTypeTest.java

```diff
@@ -129,14 +129,14 @@ public void testConversionToPointType() {
     public void testConversionToXY() {
         HSBType hsb = new HSBType("220,90,50");
         PercentType[] xy = hsb.toXY();
-        assertEquals(new PercentType("16.969364"), xy[0]);
-        assertEquals(new PercentType("12.379659"), xy[1]);
+        assertEquals(14.65, xy[0].doubleValue(), 0.01);
+        assertEquals(11.56, xy[1].doubleValue(), 0.01);
```

Comment:
These colors are nearly the same. Using e.g. https://www.easyrgb.com/en/convert.php#inputFORM to convert both colors to HTML color code results in `#021645` and `#001849` which is very close.